Stanwood_Dialog_iOS/Classes/RatingDialog.swift

```diff
@@ -13,15 +13,17 @@ import UIKit
     func timeout()
 }
 
-@objc public class RatingDialog: NSObject, RatingDialogPresenting {
+@available(iOS 10.0, *)
+@objc
+public class RatingDialog: NSObject, RatingDialogPresenting {
     
     private var text1: String?
     private var text2: String?
     private var text3: String?
     private var text4: String?
     private var faceURL: URL?
     private var bannerURL: URL?
-    private var appStoreURL: URL?
+    private var appStoreURL: URL!
```

Comment:
@epeschard Why force unwrapping?

Reply:
I was trying to remove the force unwrapping bellow now done properly with optional binding